.github/workflows/release_python.yml

```diff
@@ -304,6 +315,10 @@ jobs:
         #   repository-url: https://test.pypi.org/legacy/
 
       - name: Publish release
+        # We don't need the source for the release, so we don't currently check it out.
```

Comment:
This should fix the failure we observed @jordanrfrazier. We should probably do an alpha release soon after this goes in to verify it works / flush out any surprises.